x-pack/test/functional/apps/grok_debugger/grok_debugger.js

```diff
@@ -13,7 +13,7 @@ export default function ({ getService, getPageObjects }) {
   const PageObjects = getPageObjects(['grokDebugger']);
 
   // FLAKY: https://github.com/elastic/kibana/issues/84440
```

Comment:
nit: can you remove this comment if the issue has been resolved?

Reply:
Removed.